### DISCUSSION TITLE
Header height

### DISCUSSION THREAD

**Discussion by urigross** [elementor/elementor#22023] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I need to set header height if I don't set the logo.

### Describe the solution you'd like

I could set the header height

**Comment by ayoubkhan558** [comment]
you can set is using custom custom CSS.

**Comment by loucasl** [comment]
This is what the min height setting is for on your container / section
